### PR TITLE
[WIP] Transaction_View SendSagaEvent Compatibility

### PR DESCRIPTION
**Pull request by TimWDev**

### CODE REVIEW

File: actors/transaction_view/src/main.rs [hunk 1]
```diff
@@ -248,6 +249,37 @@ pub fn process_event(v: TopicPushEnvelope) -> () {
     }
 }
 
+#[update]
+pub fn process_saga_event(v: TopicPushEnvelope) -> () {
```
Comment (TimWDev) [inline actors/transaction_view/src/main.rs:253]: Reminder to self: add Candid interface entry for this new endpoint

File: src/rs_quark/send_saga/channel_handlers.rs [hunk 2]
```diff
@@ -8,18 +8,22 @@ use crate::send_saga::send_saga_data::{SendSagaData, SendSagaState};
 use candid::CandidType;
 use serde::{Deserialize, Serialize};
 
+fn clock() -> u64 {
```
Comment (TimWDev) [inline src/rs_quark/send_saga/channel_handlers.rs:11]: Self reminder: can remove this now

File: actors/send_saga/src/main.rs [hunk 3]
```diff
@@ -393,7 +398,8 @@ fn process_event(event: SendSagaEvent) -> Result<(), String> {
             let command = StartSagaCommand {};
             SAGA_DATA_CACHE.with(|data| match data.borrow_mut().get_mut(&v.id) {
                 Some(d) => match command.execute_on(&d) {
-                    Ok(x) => {
+                    Ok(mut x) => {
```
Comment (SuddenlyHazel) [inline actors/send_saga/src/main.rs:401]: Nit, can we give this variable x a descriptive name?

File: actors/send_saga/src/main.rs [hunk 4]
```diff
@@ -522,7 +528,7 @@ pub fn handle_account_event(
                                 Some(apply_id) => {
                                     match data.get_mut(&apply_id) {
                                         Some(d) => {
```
Comment (SuddenlyHazel) [inline actors/send_saga/src/main.rs:530]: same comment here.

**Review by SuddenlyHazel — APPROVED** [review]
Couple of nits. But, this LGTM.

**Pull request merged**